pb/p2pd.proto

```diff
@@ -59,7 +59,7 @@ message StreamOpenRequest {
 }
 
 message StreamHandlerRequest {
-  required string path = 1;
+  repeated bytes addrs = 1;
```

Comment:
i *think* stream handlers should still just accept a single address. i guess the client should really only listen on one.

Reply:
the daemon however can listen on many to give users a choice

Reply:
to be clear: a client can listen on more than one address, but should only send one of those when registering a handler

Reply:
Switched from repeated to required